### PR TITLE
style: changed dark mode background color

### PR DESCRIPTION
**Pull request by IdanDover**
# Description
I changed the bg color in dark mode to be a bit nicer specified in issue #356

If you want a different color just say so

## screenshots
-before
image

-after
image

### CODE REVIEW

**Review by shootermv — COMMENTED** [review]
i can see lot of lint issues...
may be you can run `npm run lint:fix` to solve them

**Comment by IdanDover** [comment]
yeah I did that on the last commit now it shows it passed the lint test

File: src/layout/header/GitHubLink/GitHubLink.scss [hunk 1]
```diff
@@ -2,11 +2,11 @@
   display: flex;
   justify-content: center;
   align-items: center;
-  font-size: 1.5em; 
+  font-size: 1.5em;
   cursor: pointer;
-
+  transition: all 0.1s;
```
Comment (shootermv) [inline src/layout/header/GitHubLink/GitHubLink.scss:7]: question: why do we need this transition rule?

Comment (IdanDover) [reply src/layout/header/GitHubLink/GitHubLink.scss:7]: I just wanted to add a bit of flavor when you hover over the github logo or the bulb.

**Comment by shootermv** [comment]
Lgtm

**Pull request merged**